hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/TestLargeBlockReport.java

```diff
@@ -75,6 +75,7 @@ public void tearDown() {
 
   @Test
   public void testBlockReportExceedsLengthLimit() throws Exception {
+    conf.setInt(IPC_MAXIMUM_DATA_LENGTH, IPC_MAXIMUM_DATA_LENGTH_DEFAULT / 2);
```

Comment:
Why should you set this property? Could you add a comment to explain?

Reply:
Yes,  I will add a comment.